src/main/java/eu/dissco/core/digitalspecimenprocessor/domain/DigitalMediaEvent.java

```diff
@@ -2,8 +2,8 @@
 
 import java.util.List;
 
-public record DigitalMediaObjectEvent(
+public record DigitalMediaEvent(
     List<String> enrichmentList,
-    DigitalMediaObjectWrapper digitalMediaObjectWrapper) {
+    DigitalMediaWrapper digitalMediaWrapper) {
```

Comment:
Do we still need the wrappers? The back end looks so much nicer without them...

Reply:
We have lots of wrappers/event classes, it would be nice if we could combine some of them at some point. beyond scope for now though

Reply:
Created a task for this so we don't forget 
https://naturalis.atlassian.net/browse/DD-1267